### PR TITLE
french transalation review Alain Njike

### PR DESCRIPTION
**Pull request by njikejr**
find her my review of the French translation

### CODE REVIEW

**Comment by gernotstarke** [comment]
Alain @njikejr I try to get the original translator of the FR version, Damien Lucas @dlucasd to review your proposals.

Maybe you two could come to a conclusion :-)

File: FR/asciidoc/src/08_concepts.adoc [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 ****
 .Contenu
-Cette section décrit les réglementations générales, principales et les idées de solutions qui sont pertinentes dans plusieurs parties (= transversales) de votre système.
+Cette section décrit les réglementations générales, principales et les idées de solutions qui sont importantes dans plusieurs parties (= transversales) de votre système.
 Ces concepts sont souvent liés à plusieurs briques.
 Ils peuvent inclure de nombreux sujets différents, tels que
 
 * les modèles, particulièrement les modèles de domaine
-* les patrons d'architecture ou de conception
+* les modèles d'architecture ou de conception
```
Comment (dlucasd) [inline FR/asciidoc/src/08_concepts.adoc:15]: I'm ok with this fix but if we talk about "design pattern", in french, we usually say "patron de conception". But when we talk about architecture, "modèle" is the best in fact.

File: FR/asciidoc/src/10_quality_requirements.adoc [hunk 2]
```diff
@@ -1,25 +1,25 @@
 ifndef::imagesdir[:imagesdir: ../images]
 
 [[section-quality-scenarios]]
-== Exigences de qualité
+== Critères de qualité
```
Comment (dlucasd) [inline FR/asciidoc/src/10_quality_requirements.adoc:4]: Nice, I knew there was something wrong with my translation :D

**Review by dlucasd — COMMENTED** [review]
Thank you @njikejr for this corrections. It's sound really good now!

**Review by gernotstarke — APPROVED** [review]
merci!!

**Pull request merged**